src/criteria_assessment/criteria.jl

```diff
@@ -148,6 +151,35 @@ function setup_region_routes(config, auth)
         return file(mask_path; headers=COG_HEADERS)
     end
 
+    @get auth("/suitability/assess/{reg}/{rtype}") function (
+        req::Request, reg::String, rtype::String
+    )
+        # somewhere:8000/suitability/assess/region-name/reeftype?criteria_names=Depth,Slope&lb=-9.0,0.0&ub=-2.0,40
+        # 127.0.0.1:8000/suitability/assess/Cairns-Cooktown/slopes?Depth=-4.0:-2.0&Slope=0.0:40.0&Rugosity=0.0:6.0
+
+        qp = queryparams(req)
+        @debug "In region assessment route"
```

Comment:
delete debug. If we want this kind of thing, we could add debugging middleware that logs the request before it's handled. let me know if that would be useful.